services/app-api/utils/formTemplates/formTemplates.ts

```diff
@@ -323,3 +327,21 @@ const makePCCMTemplateModifications = (reportTemplate: ReportJson) => {
   }
   programTypeQuestion.props!.disabled = true;
 };
+
+const generateTemplateWithoutIlos = (originalReportTemplate: any) => {
+  const reportTemplate = structuredClone(originalReportTemplate);
+
+  // remove ILOS sections from template
+  reportTemplate.routes = reportTemplate.routes.map((route: ReportRoute) => {
+    if (
+      route.path === "/mcpar/program-information" ||
+      route.path === "/mcpar/plan-level-indicators"
+    ) {
+      const updatedChildren = route.children?.slice(0, -1);
+      route.children = updatedChildren;
+    }
+    return route;
+  });
```

Comment:
IDEA: I think I would use `for..of` here. We're already working within a structuredClone, so there's no harm in skipping the map to do these updates in place.

```suggestion
  for (let route of reportTemplate.routes) {
    if (route.path === "/mcpar/program-information" ||
      route.path === "/mcpar/plan-level-indicators"
    ) {
      // These sections' last subsection is ILOS-specific; remove it.
      route.children = route.children?.slice(0, -1);
    }
  }
```